### PR TITLE
[Jellyfin] Overhaul README.md

### PR DESCRIPTION
**Pull request by felixschndr**
# Overhaul Jellyfin README

Just some simple changes to the README
- Fix some typos
- Add some code brackets
- Some formatting changes

### CODE REVIEW

File: bundles/org.openhab.binding.jellyfin/README.md [hunk 1]
```diff
@@ -70,20 +61,27 @@ In order to assist you with this process the binding expose a simple login form
...
 The default behavior will look for movies, series, or episodes whose name start with the provided text, if it found results the prevalence go as said before.
 If the result is a series the binding will try to resume some episode, if not it will look for the next episode to watch and finally will fall back to the first episode.
 
-You can prefix your search with '\<type:movie\>', '\<type:episode\>', '\<type:series\>' to restrict your search to a given type.
+You can prefix your search with `<type:movie>`, `<type:episode>`, `<type:series>` to restrict your search to a given type.
 
-Also, you can target a specific series episode by season and episode numbers prefixing your search with '\<season:1\>\<episode:1\>' with the desired values. So '\<season:3\>\<episode:10\>Something' will try to play the episode 10 for the season 3 of the series named 'Something'.
+Also, you can target a specific series episode by season and episode numbers prefixing your search with `<season:1><episode:1>` with the desired values. So `<season:3><episode:10>Something` will try to play the episode `10` for the season `3` of the series named `Something`.
+
+## Known limitations
```
Comment (jlaur) [inline bundles/org.openhab.binding.jellyfin/README.md:75]:
```suggestion
## Known Limitations
```

File: bundles/org.openhab.binding.jellyfin/README.md [hunk 2]
```diff
@@ -70,20 +61,27 @@ In order to assist you with this process the binding expose a simple login form
...
+
+This binding was tested against the android tv, and web clients.
```
Comment (jlaur) [inline bundles/org.openhab.binding.jellyfin/README.md:77]:
```suggestion
This binding was tested against Android TV and web clients.
```

I think?

File: bundles/org.openhab.binding.jellyfin/README.md [hunk 3]
```diff
@@ -70,20 +61,27 @@ In order to assist you with this process the binding expose a simple login form
...
+The only problem that were found is that the channels `play-next-by-terms` and `play-last-by-terms` don't work currently on the android tv client.
```
Comment (jlaur) [inline bundles/org.openhab.binding.jellyfin/README.md:78]:
```suggestion
The only problem that was found is that the channels `play-next-by-terms` and `play-last-by-terms` don't currently work on the Android TV client.
```

File: bundles/org.openhab.binding.jellyfin/README.md [hunk 4]
```diff
@@ -70,20 +61,27 @@ In order to assist you with this process the binding expose a simple login form
...
+
+Before open an issue please test you are able to correctly control your device from the Jellyfin web ui to identify whetter is an issue on the client itself.
```
Comment (jlaur) [inline bundles/org.openhab.binding.jellyfin/README.md:80]:
```suggestion
Before opening an issue, please test that you are able to correctly control your device from the Jellyfin web UI to identify whether it's an issue on client side.
```

File: bundles/org.openhab.binding.jellyfin/README.md [hunk 5]
```diff
@@ -97,35 +95,35 @@ Bridge jellyfin:server:exampleServerId "Jellyfin Server" [
...
+### Example Client - jellyfin_clients.things
 
 ```java
-Thing jellyfin:client:exampleServerId:<JELLYFIN_DEVICE_ID> "Jellyfin Web client" (jellyfin:server:exampleServerId)
+Thing jellyfin:client:exampleServerId:<JELLYFIN_DEVICE_ID> "Jellyfin Web client"     (jellyfin:server:exampleServerId)
 Thing jellyfin:client:exampleServerId:<JELLYFIN_DEVICE_ID> "Jellyfin Android client" (jellyfin:server:exampleServerId)
 ```
 
-- I recommend creating the clients using the discovery. For getting the device ids manually I recommend to use the Jellyfin web interface with the web inspector and look for the request that is launched when you click the cast button (<jellyfin url>/Sessions?ControllableByUserId=XXXXXXXXXXXX).
+It is recommended creating the clients using the discovery. For getting the device ids manually it is possible to use the Jellyfin web interface with the web inspector and look for the request that is launched when you click the cast button (`<jellyfin url>/Sessions?ControllableByUserId=XXXXXXXXXXXX`).
```
Comment (jlaur) [inline bundles/org.openhab.binding.jellyfin/README.md:107]:
```suggestion
It is recommended to create the clients using the discovery.
For getting the device ids manually, it is possible to use the Jellyfin web interface with the web inspector and look for the request that is launched when you click the cast button (`<jellyfin url>/Sessions?ControllableByUserId=XXXXXXXXXXXX`).
```

**Review by jlaur — CHANGES_REQUESTED** [review]
Thanks for improving the documentation. Only some minor corrections from proof-reading (please note English is not my first language, so I can also have made mistakes).

**Comment by felixschndr** [comment]
@jlaur thanks for the review. Indeed some sentences don't sounds particularly good. I just removed the `I` phrases from the README without any rewriting. 

I will take a closer look at this on the weekend

**Comment by felixschndr** [comment]
@jlaur I checked the whole README again and found some typos, wrong grammar, etc. Did some overhauling, what do you think?

File: bundles/org.openhab.binding.jellyfin/README.md [hunk 6]
```diff
@@ -1,52 +1,43 @@
...
-
-Before open an issue please test you are able to correctly control your device from the Jellyfin web ui to identify whetter is an issue on the client itself.
+This is the binding for [Jellyfin](https://jellyfin.org), the volunteer built media solution that puts you in control of your media.
+This binding allows connect to Jellyfin clients that supports remote control, it's built on top of the official Jellyfin kotlin sdk.
+Compatible with Jellyfin servers from version `10.8.1`, recommended is `10.8.13`.
 
 ## Discovery
 
-Before you are able to discover clients you should have a bridge to the server so until one is online the discovery will only look for servers on your local network. Once one is online the discovery will detect controllable clients connected to that server.
+To discover clients, you must first configure a server (bridge). After that, device discovery will detect controllable clients.
```
Comment (jlaur) [inline bundles/org.openhab.binding.jellyfin/README.md:9]:
```suggestion
To discover clients, you must first configure a server (bridge).
After that, device discovery will detect controllable clients.
```

File: bundles/org.openhab.binding.jellyfin/README.md [hunk 7]
```diff
@@ -70,20 +61,27 @@ In order to assist you with this process the binding expose a simple login form
 | play-last-by-id            | String | Add to playback queue as last by id, works for series, episodes and movies                                      |
 | browse-by-id               | String | Browse media by id, works for series, episodes and movies                                                       |
 
-### Terms search:
+### Terms Search
+
+The terms search has a default behavior that can be modified by sending some predefined prefixes.
+
+The default behavior is to search for movies, series or episodes whose name starts with the given text, if it finds results the bind will proceed as said before.
```
Comment (jlaur) [inline bundles/org.openhab.binding.jellyfin/README.md:68]:
```suggestion
The default behavior is to search for movies, series or episodes whose name starts with the given text.
If it finds results, the bind will proceed as said before.
```

**Review by jlaur — CHANGES_REQUESTED** [review]
Thanks for the additional updates. I have added a few more minor comments.

**Comment by felixschndr** [comment]
I corrected the changes you requested. Some newlines you requested (like [this](https://github.com/openhab/openhab-addons/pull/16359#discussion_r1485144803)) are too much (in my opinion) so I left them as is.

**Comment by jlaur** [comment]
> I corrected the changes you requested. Some newlines you requested (like [this](https://github.com/openhab/openhab-addons/pull/16359#discussion_r1485144803)) are too much (in my opinion) so I left them as is.

Thanks! What do you mean by "too much"? I should have mentioned that one sentence per line is preferred because it will make it easier to view diffs. It doesn't impact rendering.

**Comment by felixschndr** [comment]
Looks good now @jlaur?

**Review by jlaur — APPROVED** [review]
LGTM

**Pull request merged**